lib/Rex/Group/Lookup/XML.pm

```diff
@@ -33,7 +33,7 @@ use Rex -base;
 require Exporter;
 use base qw(Exporter);
 use vars qw(@EXPORT);
-XML::LibXML->require;
+use XML::LibXML;
```

Comment:
[Official EOL of RHEL4 extended support is 2017-03-31](https://access.redhat.com/support/policy/updates/errata/#Life_Cycle_Dates). I'm fine with a Rex "policy" of only supporting up to normal production lifetime (i.e. excluding extended life-cycle).

On one part, I believe as a software project/community we should endorse best practices such as updates and upgrades done in a timely fashion regularly. On the other hand, taking on extended lifecycle support for RHEL5 would mean being stuck with lots of ballast till end of 2020. Which sounds scary.